pkg/db/migration/000014_init.up.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS public.tag(
+    uid UUID NOT NULL,
+    id VARCHAR(255) NOT NULL,
```

Comment:
Since the tag table won't be used standalone, I think we don't need another `uid` and `id` field. Keeping only `pipeline_uid`, `tag_name`, and timestamps should be enough.